packages/trader/src/AppV2/Containers/Trade/trade.scss

```diff
@@ -3,7 +3,6 @@
     flex-direction: column;
     background-color: var(--core-color-solid-slate-75);
     padding: 0 var(--core-spacing-400) var(--core-spacing-400);
-    gap: var(--core-spacing-400);
```

Comment:
if you remove gap here, it'll also remove it between trader params and chart.

Reply:
reverted . thanks